manifests/server/apiserver.pp

```diff
@@ -6,7 +6,7 @@
 
   K8s::CIDR $service_cluster_cidr = $k8s::service_cluster_cidr,
 
-  Optional[Array[Stdlib::HTTPUrl]] $etcd_servers = undef,
+  Optional[Array[Stdlib::HTTPUrl]] $etcd_servers = $k8s::etcd_servers,
```

Comment:
To keep things consistent, this should probably be copied into `k8s::server` and grabbed from there instead.